cpp/tests/CMakeLists.txt

```diff
@@ -452,9 +452,13 @@ ConfigureTest(EIGENVECTOR_CENTRALITY_TEST centrality/eigenvector_centrality_test
 # - WEAKLY CONNECTED COMPONENTS tests -------------------------------------------------------------
 ConfigureTest(WEAKLY_CONNECTED_COMPONENTS_TEST components/weakly_connected_components_test.cpp)
 
+###################################################################################################
+# - Legacy RANDOM_WALKS tests ----------------------------------------------------------------------------
+ConfigureTest(LEGACY_RANDOM_WALKS_TEST sampling/random_walks_test.cu)
+
 ###################################################################################################
 # - RANDOM_WALKS tests ----------------------------------------------------------------------------
-ConfigureTest(RANDOM_WALKS_TEST sampling/random_walks_test.cu)
+ConfigureTest(RANDOM_WALKS_TEST sampling/sg_random_walks_test.cu)
```

Comment:
Should we name SG tests with `sg_`? Our convention has been omitting `sg_` for SG tests. If we decide to use `sg_` for SG tests, we need to apply this for all SG tests.

Reply:
This is also temporary.  There is an existing random walks test for the original implementation (I hesitate to use the word legacy, since we usually refer to implementations that use the legacy graph objects).  I can't delete the original test until I have a replacement working.

I could rename the existing as legacy if you think that would be cleaner.

Reply:
OK, no complaint if temporary, just don't forget to fix this later.

Reply:
Added a FIXME to remind me